ttnn/cpp/ttnn/operations/moreh/moreh_arange/device/moreh_arange_device_operation.hpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <variant>
+#include <vector>
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/device_operation.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+#include "ttnn/tensor/types.hpp"
+
+namespace ttnn::operations::moreh::moreh_arange {
+
+struct MorehArangeOperation {
+    struct operation_attributes_t {
+        float start;
+        float end;
+        float step;
+        bool untilize_out;
+        const std::optional<DataType> output_dtype;
+        const std::optional<MemoryConfig> output_memory_config;
```

Comment:
```suggestion
        const MemoryConfig output_memory_config;
```
 Inside operation_attributes_t, it’s MemoryConfig.

Reply:
I changed it as TT's method.